src/App.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Switch, BrowserRouter as Router, Route } from 'react-router-dom';
+import ForConsumers from './pages/ForConsumers';
```

Comment:
no unused variables

Reply:
yuoo need to fix your eslint rules properly, to make sure you are getting the errors